clojure_liberator/src/clojure_liberator/core.clj

```diff
@@ -21,6 +21,20 @@
    :type       type-name
    :attributes (dissoc value :id)})
 
+(defn- update-vals [map vals f]
+  (reduce #(update-in %1 [%2] f) map vals))
+
+(defn- parse-date [value]
```

Comment:
I didn't want to move the helper functions to another namespace until I
needed to use them outside of this one. Then, it would be the first choice,
for sure.